csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -778,27 +785,50 @@ static Settings()
             /// </summary>
             public TypeRegistry TypeRegistry { get; }
 
+            /// <summary>
+            /// Whether to format enums as ints. Defaults to false.
+            /// </summary>
+            public bool FormatEnumsAsIntegers { get; }
+
             // TODO: Work out how we're going to scale this to multiple settings. "WithXyz" methods?
 
             /// <summary>
             /// Creates a new <see cref="Settings"/> object with the specified formatting of default values
-            /// and an empty type registry.
+            /// and an empty type registry. This constructor is deprecated.
```

Comment:
I don't think we really need to deprecate these constructors, to be honest - there's nothing wrong with them, really. (I don't think we want to end up adding warnings for entirely valid code which will keep working.)